### ISSUE TITLE
RuntimeError with manuskriptw.exe (sys.stderr is None)

### ISSUE THREAD

**Issue by Anne501R** [olivierkes/manuskript#1138]
Hi, I installed manuskript 0.15.0 on Win11. I tried to run manuskriptw.exe, but get the error: _Failed to execute script 'manuskript' due to unhandled exception: sys.stderr is None_

```
Traceback (most recent call last):
File "manuskript", line 11, in <module>
File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
File "manuskript\main.py", line 18, in <module>
RuntimeError: sys.stderr is None
```

The Traceback looks a lot like the one in  #1121. manuskript.exe runs without problems.

**Comment by TheJackiMonster** [comment]
Thank you for reporting.

**Comment by SpeedyDoggo** [comment]
Getting the same issue on Windows 10, if that helps at all.

**Comment by Dreaded-Gnu** [comment]
I've played a bit around with windows builds of manuskript. Finally the error comes from `faulthandler.enable()`. Proposal for a fix would be to enable faulthandler only when executable is not manuskriptw like in the following snippet:
```
try:
    if "manuskriptw" not in sys.executable:
        faulthandler.enable()
except AttributeError:
    print("Faulthandler failed")
```
In case such a fix would be acceptable I would provide a pull request for it.

**Comment by TheJackiMonster** [comment]
The next release should solve the issue. @Dreaded-Gnu Thanks for the contribution!

**Comment by lmiol** [comment]
between 0.15.0 and 0.14.0 versions is 1 year.
Could you merge fixes of critical errors, like this one into 0.15.1 version? Regards
